From: Andrew Morton <akpm@linux-foundation.org>
Subject: selftests-mm-skip-hugetlb_dio-tests-when-dio-alignment-is-incompatible-checkpatch-fixes
Date: Tue Mar 31 04:51:23 PM PDT 2026

ERROR: code indent should use tabs where possible
#94: FILE: tools/testing/selftests/mm/hugetlb_dio.c:48:
+^I^I^I        unsigned int end_off, unsigned int align)$

Cc: David Hildenbrand (Arm) <david@kernel.org>
Cc: Li Wang <liwang@redhat.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 tools/testing/selftests/mm/hugetlb_dio.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/tools/testing/selftests/mm/hugetlb_dio.c~selftests-mm-skip-hugetlb_dio-tests-when-dio-alignment-is-incompatible-checkpatch-fixes
+++ a/tools/testing/selftests/mm/hugetlb_dio.c
@@ -45,7 +45,7 @@ static int get_dio_alignment(int fd)
 }
 
 static bool check_dio_alignment(unsigned int start_off,
-			        unsigned int end_off, unsigned int align)
+				unsigned int end_off, unsigned int align)
 {
 	unsigned int writesize = end_off - start_off;
 
_
